### PR TITLE
Update translation

### PR DESCRIPTION
**Pull request by DarkThinking**
**Bug fix?**
no

**New Feature?**
idk

**Resolve an issue?**
no

### CODE REVIEW

**Review by djhvscf — APPROVED** [review]

**Comment by djhvscf** [comment]
Please review CI errors

**Review by djhvscf — CHANGES_REQUESTED** [review]
Please review CI errors

**Comment by UtechtDustin** [comment]
@DarkThinking Could you fix the CI errors, so we can merge it :-) ?

**Comment by djhvscf** [comment]
Hi! There is not any reply, in that case we are going to close this PR. Thanks!

File: src/locale/bootstrap-table-es-ES.js [hunk 1]
```diff
@@ -48,43 +48,43 @@ $.fn.bootstrapTable.locales['es-ES'] = $.fn.bootstrapTable.locales['es'] = {
...
   formatColumns () {
     return 'Columnas'
   },
   formatColumnsToggleAll () {
-    return 'Toggle all'
+    return 'Cambiar todo'
   },
   formatFullscreen () {
-    return 'Fullscreen'
+    return 'Pantalla completa
```
Comment (wenzhixin) [inline src/locale/bootstrap-table-es-ES.js:75]: `'Pantalla completa'`

File: src/locale/bootstrap-table-es-ES.js [hunk 2]
```diff
@@ -48,43 +48,43 @@ $.fn.bootstrapTable.locales['es-ES'] = $.fn.bootstrapTable.locales['es'] = {
...
   },
   formatAllRows () {
     return 'Todos'
   },
   formatAutoRefresh () {
-    return 'Auto Refresh'
+    return 'Auto Recargar
```
Comment (wenzhixin) [inline src/locale/bootstrap-table-es-ES.js:81]: `'Auto Recargar'`

File: src/locale/bootstrap-table-es-ES.js [hunk 3]
```diff
@@ -48,43 +48,43 @@ $.fn.bootstrapTable.locales['es-ES'] = $.fn.bootstrapTable.locales['es'] = {
...
   },
   formatExport () {
     return 'Exportar los datos'
   },
   formatJumpTo () {
-    return 'GO'
+    return 'IR
```
Comment (wenzhixin) [inline src/locale/bootstrap-table-es-ES.js:87]: `'IR'`

**Pull request closed without merging**